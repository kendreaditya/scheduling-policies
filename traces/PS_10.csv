1,0,10
2,2,7
3,4,6
4,6,5
5,8,2
6,10,1
7,12,3
8,14,5
9,16,7
10,18,9
11,20,11
12,22,13
13,24,15
14,26,17
15,28,19
16,30,21
17,32,23
18,34,25
19,36,27
20,38,29
