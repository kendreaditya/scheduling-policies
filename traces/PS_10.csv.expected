1,94
2,81
3,81
4,65
5,30
6,16
7,48
8,81
9,118
10,140
11,160
12,178
13,194
14,208
15,220
16,235
17,243
18,249
19,253
20,255
